xml/adm_wbem.xml

```diff
@@ -1898,7 +1898,7 @@ localhost:5988/root/cimv2:Linux_Ext4FileSystem FSReservedCapacity=, \
     </listitem>
    </varlistentry>
    <varlistentry>
-    <term><link xlink:href="http://sblim.sourceforge.net/wiki/index.php/Main_Page"/>
+    <term><link xlink:href="http://sblim.sourceforge.net/wiki/index.php/main_Page"/>
```

Comment:
Changing case within links may be critical, too. I checked: in this case, the link still works but it took long to load.

Reply:
Good catch, I think that is fixable though.